### PR TITLE
[MB-2105] List all matchable attributes

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

**Comment by katrogan** [comment]
@anandswaminathan

**Comment by katrogan** [comment]
ping @anandswaminathan @EngHabu

File: pkg/repositories/gormimpl/resource_repo.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ type ResourceRepo struct {
 	metrics          gormMetrics
 }
 
+const priorityDescending = "priority desc"
```
Comment (EngHabu) [inline pkg/repositories/gormimpl/resource_repo.go]: nit: Is there an `ID.Priority` that has the `priority` string to use here?

Comment (katrogan) [reply pkg/repositories/gormimpl/resource_repo.go]: from the docs it looks like you still have to explicitly use the string field name :\

on the plus side, model fields should change infrequently :-)

**Pull request merged**